src/context/service-worker-context.tsx

```diff
@@ -32,7 +72,7 @@ export const ServiceWorkerProvider = ({ children }): JSX.Element => {
       }
     }
     void doWork()
-  }, [])
+  }, [isServiceWorkerRegistered])
```

Comment:
we're dependent upon `isServiceWorkerRegistered`, so we need to execute this effect more than a single time now